go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/zyedidia/clipper v0.1.1
 	github.com/zyedidia/glob v0.0.0-20170209203856-dd4023a66dc3
 	github.com/zyedidia/json5 v0.0.0-20200102012142-2da050b1a98d
-	github.com/zyedidia/tcell/v2 v2.0.10-0.20230831153116-061c5b2c7260 // indirect
+	github.com/zyedidia/tcell/v2 v2.0.10-0.20231016092200-969958742a55 // indirect
```

Comment:
Generally it's better to do it in a commit before the commit that depends on it, not after it (to ensure that no single commit has a regression). Although maybe it our case it is not so important, since @zyedidia tends to merge PRs as a single squashed commit anyway...

Reply:
Rebased and changed the order for the sake of completeness. ;)